packages/block-editor/src/components/link-control/link-preview.js

```diff
@@ -68,6 +68,7 @@ export default function LinkPreview( {
 				'is-preview': true,
 				'is-error': isEmptyURL,
 			} ) }
+			role="option"
```

Comment:
I'm not exactly sure what's going on in this part of the component, because `<LinkPreview>` never seems to be [contained in a `listbox`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/option_role#associated_aria_roles_states_and_properties). Could it be the `aria-selected` part that's unnecessary, perhaps?

Reply:
Good call. I've taken a look at the component in action and it doesn't seem to behave as an option in a listbox either. 

I've removed the role and the `aria-selected` as you suggested and fixed the test in https://github.com/WordPress/gutenberg/pull/43279/commits/c14c0c6b12597c975f1220e597c20c121949c981. 

Feel free to take another look when you get a chance.